packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.WrapAdvanced.Example.tsx

```diff
@@ -32,18 +32,17 @@ export class HorizontalStackWrapAdvancedExample extends React.Component<{}, IExa
       root: {
         background: DefaultPalette.themeTertiary,
         width: `${stackWidth}%`,
```

Comment:
@khmakoto this here is an anti-pattern. You're going to mutate the width of a thing, which will cause re-evaluation of the styles and a re-calculation of classnames. A better approach is to use inline styling for constantly mutating widths or positions. That way you keep the classname generation minimized to static content.